src/lib.rs

```diff
@@ -160,6 +160,9 @@
 
 #![deny(warnings, missing_docs, missing_debug_implementations)]
 
+// Nightly Regression https://github.com/rust-lang/rust/issues/70814
+#![allow(unknown_lints, unused_braces)]
```

Comment:
The issue that was causing the nightly build to fail (issue 70814) may have been fixed (or at least have had a work around added to rustc) in nightly-2020-04-26. We should remove this change before merging to see if nightly is still broken.

Reply:
I reverted this change and the pull request has built on nightly so this build breakage is resolved.